extensions/wikia/Rail/styles/RailElement.scss

```diff
@@ -0,0 +1,269 @@
+@import "skins/shared/color";
+
+$black: #000;
+$white: #FFF;
+$color-divider: #E6EBF2;
+$color-rail-background: mix($color-page, $black, 90%);
+@if $is-dark-wiki {
+    $color-rail-background: mix($color-page, $white, 90%);
+}
+
+
+@mixin wikia-rail-element() {
+    background-color: $color-rail-background;
+    padding: 20px;
+
+    zoom: 1;
+    margin-bottom: 10px;
+    position: relative;
+    word-wrap: break-word;
+}
+
+.wikia-rail {
+
+    float: right;
+    overflow: hidden;
+    padding: 0 10px 10px 0;
+    width: 300px;
+
+    h1 {
+        padding-left: 0;
+    }
+
+    .WikiaActivityModule {
+
+        @include wikia-rail-element;
+        font-size: 12px;
+        font-weight: bold;
+
+        a {
+            color: $color-links-default;
+        }
+
+        a:hover {
+            color: $color-links-default;
+            text-decoration: underline;
+            text-shadow: initial;
+        }
+
+        .activity-heading {
+            font-size: 24px;
+            font-weight: bold;
+            margin-bottom: 20px;
+        }
+
+        li em a {
+            font-size: 16px;
+            font-weight: bold;
+            margin-top: 20px;
+        }
+
+        .more {
+            float: right;
+            font-size: 12px;
+            font-weight: bold;
+            margin-top: 30px;
+            text-transform: uppercase;
+        }
+
+        .edited-by {
+            font-size: 12px;
+            font-weight: normal;
+            margin-top: 10px;
+
+            a {
+                font-size: 12px;
+            }
+        }
+
+        .sprite {
+            left: 3px;
+            position: absolute;
+            top: 8px;
+        }
+    }
+
+    .videos-module-rail {
+
+        @include wikia-rail-element;
+        display: block;
+
+        h1 {
+            font-size: 24px;
+            font-weight: bold;
+            margin-bottom: 16px;
+        }
+
+
+        .thumbnails {
+
+            a:first-of-type {
+                color: $color-links;
+                margin-bottom: 0;
+            }
+
+            a:first-of-type:hover {
+                color: $color-links;
+                text-decoration: underline;
+                text-shadow: initial;
+            }
+
+            .title {
+                font-size: 12px;
+                height: 40px;
+                margin-top: 10px;
+
+                a {
+                    display: block;
+                    margin: 0;
+                    overflow: hidden;
+                    text-overflow: ellipsis;
+                    white-space: nowrap;
+                    word-wrap: normal;
+                }
+            }
+        }
+    }
+
+    &.new-chat {
+        .ChatModule {
+
+            @include wikia-rail-element;
+
+            p {
+                color: mix($color-text, $color-page, 60%);
+                font-size: 12px;
+            }
+
+            .chat-headline {
+
+                .chat-live {
```

Comment:
the more class I add on the path the stronger the condition will be and will override old .WikiaRail class.